docs/source/installation/troubleshooting.rst

```diff
@@ -2,3 +2,23 @@ Troubleshooting
 ===============
 
 List of known installation problems.
+
+Some letters are missing from TextMobject/TexMobject output?
+------------------------------------------------------------
+
+If you have recently installed TeX you may need to build the fonts it
+uses. Which can be done by running:
+
+.. code-block:: bash
+
+  fmtutil -sys --all
+
+The "LaTeX" in the OpeningManimExample scene displays incorectly.
+-----------------------------------------------------------------
+
+If you are on a Ubuntu/Debian system this may be caused by missing python
+bindings for Cairo. Installing the ``python3-gi-cairo`` package can solve this:
+
+.. code-block:: bash
+
+  sudo apt install python3-gi-cairo
```

Comment:
I don't think this is an issue. Because the users don't require to install pygobject to run manim and I think this won't cause a problem.